Marlin/src/pins/stm32f4/pins_MELLOW_FLY_E3_V2.h

```diff
@@ -96,10 +96,8 @@
 //
 // Probe enable
 //
-#if ENABLED(PROBE_ENABLE_DISABLE)
-  #ifndef PROBE_ENABLE_PIN
-    #define PROBE_ENABLE_PIN          SERVO0_PIN
-  #endif
+#if ENABLED(PROBE_ENABLE_DISABLE) && !defined(PROBE_ENABLE_PIN)
+  #define PROBE_ENABLE_PIN          SERVO0_PIN
 #endif
```

Comment:
Changed to match every other pins file that defines the `PROBE_ENABLE_PIN` this way.